projects/plugins/jetpack/modules/blocks.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Module Name: Blocks
+ * Module Description: Add additional blocks to your site and post editors.
+ * Sort Order: 5
+ * First Introduced: 13.9
+ * Requires Connection: Yes
```

Comment:
I don't think those should require a connection, since some of those blocks can be used when in offline mode, and since we do the necessary connection checks within each block's registration function.

Reply:
👍🏼  removed in 7b88491851da4104095631d680ad2933cb717576

Reply:
Did you forget to push?